### ISSUE TITLE
Store and use class labels in MLPClassifier

### ISSUE THREAD

**Issue by jlewi** [kubeflow/code-intelligence#83]
It looks like our repo_specific_model is using MLPClassifier.
https://scikit-learn.org/stable/modules/generated/sklearn.neural_network.MLPClassifier.html

The labels, however, are not stored in MLPClassifier's classes attribute.
Instead the label's are stored in the YAML file describing the model.

The code assumes that the labels in the YAML file are listed in the same order as the probabilities generated by the MLPClassifier for predictions.

It would probably be better to store the labels in the MLPClassifier to decrease the risk of them getting out of sync.
